scripts/clean_appliance.py

```diff
@@ -41,7 +41,7 @@ def main():
     }
     if args.hostname is not None:
         ssh_kwargs['hostname'] = args.hostname
-        appliance = IPAppliance(args.hostname)
+        appliance = RegularAppliance(args.hostname)
         if appliance.version < '5.9':
             # Running evm:db:reset on CFME 5.8 sometimes leaves it in a state
```

Comment:
Do we still need to consider appliance versions below "5.9"?

Reply:
No - but we can clean this up in a followup, thanks @ganeshhubale !